src/sage/groups/perm_gps/permgroup.py

```diff
@@ -1365,6 +1365,29 @@ def ngens(self):
         """
         return len(self.gens())
 
+    def is_trivial(self):
+        r"""
+        Return ``True`` if this group is the trivial group.
+
+        A permutation group is trivial, if it consists only of the
+        identity element, that is, if it has no generators.
```

Comment:
This method is a very welcome addition, thanks. The code is good as far as I understand. A small complaint about the docstring: In Sage a trivial group may in fact have a (trivial) generator. To my surprise, here we specify no generators but end up with one generator anyway:
```
sage: PermutationGroup([],domain=[2]).gens()
((),)
```
If the docstring mentions generators (and I think it is a good idea, since it clarifies to the user what the code will actuall be checking), it should mention this surprising possibility.

(I believe/hope it is not possible to make a group with *several* trivial generators, at least not easily. If that is a possibility, we would need to check that *all* generators are trivial.)

Reply:
It is, actually.

Reply:
Oh well, it is possible and not that difficult...
```
sage: P=PermutationGroup([(),()], canonicalize=False)
sage: P.gens()
((), ())
sage: P.order()
1
```

A human would likely not create such a group on purpose, but I imagine some code might. Anyway it is possible. Mathematically of course this is a trivial group, so Sage should recognize it as one.

Reply:
crossed.

Reply:
Looks good now!